src/Titanic/Titanic.jl

```diff
@@ -116,13 +115,13 @@ julia> using MLDatasets: Titanic
 julia> target = Titanic.targets();
 
 julia> summary(target)
-"1×891 Matrix{Float64}"
-     
+"1×891 Matrix{}"
+```
 """
 
```

Comment:
the should be no blank line between the function definition and its docstring

Reply:
Oohk that's what I was missing the whole time. Will fix it in a couple of hours .